Doc/c-api/init.rst

```diff
@@ -373,6 +373,14 @@ Initializing and finalizing the interpreter
    :c:func:`Py_Initialize` is called again.
 
 
+.. c:function:: int Py_IsFinalizing()
+
+   Return non-zero if the Python interpreter is :term:`shutting down
+   <interpreter shutdown>`, return 0 otherwise.
```

Comment:
Is this referring to the runtime (AKA main interpreter) or to the current interpreter?  I'd expect it to be the current interpreter.

Would it make sense to explicitly pass an interpreter and use `NULL` to mean the current one (or main)?

Reply:
I copied the documentation of the sys.is_finalizing() function. If you want a different feature (pass an interpreter), I would prefer to have a different function.

It's set by Py_Finalize() which finalizes the main interpreter: other sub-interpreters should be deleted before explicitly, no?

Reply:
I updated the doc to specify that we are talking about the **main** interpreter.